src/Resolver/DependencyResolver.php

```diff
@@ -287,7 +285,7 @@ public function resolveParameters(string $requestedType, array $callTimeParamete
 
         foreach ($params as $paramInfo) {
             $name = $paramInfo->getName();
-            $type = $paramInfo->getType();
+            $type = $paramInfo->getType() === 'self' ? $requestedType : $paramInfo->getType();
```

Comment:
We should only apply codestyle changes in this PR. 
If this would be an improvement, please create another PR with this change.

Please revert this change in this PR.

Reply:
You are right. 
At first the phpcbf changes in the X.php/Y.php  the a Typehint to self and the ```$paramInfo->getType()``` returned the string `self` instead of the fqcn. Therefore I added the ```// phpcs:ignoreFile``` to X.php/Y.php I I forgot to revert these changes.